n3fit/src/evolven3fit/eko_utils.py

```diff
@@ -228,4 +221,3 @@ def find_nf(mu, theory, thresholds):
         nf = 5
     else:
         nf = 6
-    return nf
```

Comment:
is this a typo?

Reply:
Yes.

But I wonder whether the entire thing should go (since nf0 is now given so this is never going to be used).

Reply:
then we can drop this at all